deployment/config.yaml

```diff
@@ -0,0 +1,111 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+project:
+  dev: datcom-mixer-staging # datacommons-website-dev
+  staging: datacommons-website-staging
+  prod: datacommons-website-prod
+
+image:
+  mixer: gcr.io/datcom-ci/datacommons-mixer:latest
+  website:
+    dev: website:local
+    staging: gcr.io/datcom-ci/datacommons-website:latest
+    prod: gcr.io/datcom-ci/datacommons-website:latest
+
+pull_policy:
+  dev: Never
+  staging: Always
+  prod: Always
+
+args:
+  mixer:
+    bt_project: google.com:datcom-store-dev
+    bt_instance: prophet-cache
+    branch_folder:
+      dev: "dummy" # Emtpy branch cache folder for faster loading
+      staging:
+      prod:
+  esp:
+    non_gcp:
+      dev: "--non_gcp"
+      staging:
+      prod:
+
+replicas:
```

Comment:
wow, so each website will have a dedicated mixer? i wonder if we might need more website's per mixer, but would be great to test this setup first.

Reply:
Yeah, we can study the traffic behavior see if this suits the need,